src/System.Linq.Expressions/tests/Unary/UnaryDecrementTests.cs

```diff
@@ -179,6 +180,16 @@ private static void VerifyDecrementInt(int value, bool useInterpreter)
             Assert.Equal((int)(--value), f());
         }
 
+        private static void VerifyDecrementIntMakeUnary(int value, bool useInterpreter)
+        {
+            Expression<Func<int>> e =
+                Expression.Lambda<Func<int>>(
+                    Expression.MakeUnary(ExpressionType.Decrement, Expression.Constant(value), null),
+                    Enumerable.Empty<ParameterExpression>());
+            Func<int> f = e.Compile(useInterpreter);
+            Assert.Equal((int)(--value), f());
```

Comment:
It's not, it was just copied from the test above (which I think was produced by a script, they have a lot of unnecessary casts).